src/containers/PipelineRun/PipelineRun.test.js

```diff
@@ -19,14 +19,12 @@ import configureStore from 'redux-mock-store';
 import { createIntl } from 'react-intl';
 import { PipelineRunContainer } from './PipelineRun';
 import { renderWithRouter } from '../../utils/test';
-import { messages } from '../../constants';
 
 beforeEach(jest.resetAllMocks);
 
 const intl = createIntl({
   locale: 'en',
-  defaultLocale: 'en',
-  messages: messages.en
+  defaultLocale: 'en'
```

Comment:
Tests will use the default message instead of whats in the messages ? 

I remember trying this and seeing the ID string show up

Reply:
It'll use the defaultMessage, just rebased on master and verified. Might be related to previous use of `defineMessages` too